lib/integrated-service-sdk/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Just curious: what are the pros (and cons) of defining this as an inlined Golang module of the Pipeline repository (with its own build infrastructure, license, CI) as opposed to putting it into its own repository?

I believe I can see the possible pros of the separate repository approach, but I'm failing to grasp the cons.
It could also be git-submoduled under Pipeline if that is preferred over go modding it.
(This might be simple difference in perspectives - for example I'm preferring modular approach over monolithic structures - I'm just trying to understand the reasoning behind the current solution for now.)

Are there going to be any possible consumers other than the Pipeline?